### PR TITLE
add websockets

### PR DESCRIPTION
**Pull request by jefer94**

### CODE REVIEW

File: breathecode/websocket/consumers.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+from django.contrib.auth.models import AnonymousUser
+
+MAX_INT = 1000000000000
+
+
+# @ws_auth
+# @ws_capable_of('read_student')
+# @ws_has_permission('delete_assetalias')
+@ws_can_auth
+class OnlineFromCohortConsumer(AsyncJsonWebsocketConsumer):
```
[the remark this answers is not in the document]
Comment (alesanchezr) [reply breathecode/websocket/consumers.py:17]: lets leave the online status for the other consume below, this one only will take care of the classroom activity:
- Student Reactions?
- Rising the hand?
- Classroom chat?

Note: do not implement any of this yet, it has to be discused.

File: breathecode/websocket/consumers.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+            return
+
+        await self.send_json({
+            'status': event['type'],
+            'id': event['id'] or 0,
+        })
+
+
+@ws_can_auth
+class OnlineFromAcademyConsumer(AsyncJsonWebsocketConsumer):
```
[the remark this answers is not in the document]
Comment (alesanchezr) [reply breathecode/websocket/consumers.py:88]: This consumer will take care of the onlin status for now.

File: breathecode/asgi.py [hunk 3]
```diff
@@ -9,8 +9,14 @@
 
 import os
 
+from channels.auth import AuthMiddlewareStack
```
Comment (alesanchezr) [inline breathecode/asgi.py:12]: Remember to update the Procfile to stop using unicorn.

**Review by alesanchezr — CHANGES_REQUESTED** [review]
Important changes below

**Comment by jefer94** [comment]
This PR was closed because it went crazy

**Pull request closed without merging**